kfs-core/src/main/java/edu/arizona/kfs/gl/businessobject/lookup/EntryBoHelperServiceImpl.java

```diff
@@ -0,0 +1,323 @@
+        isSameEntry &= isPropertyEqual(entry.getAccountNumber(), KFSPropertyConstants.ACCOUNT_NUMBER, columnList);
+        isSameEntry &= isPropertyEqual(entry.getSubAccountNumber(), KFSPropertyConstants.SUB_ACCOUNT_NUMBER, columnList);
+        isSameEntry &= isPropertyEqual(entry.getFinancialObjectCode(), KFSPropertyConstants.FINANCIAL_OBJECT_CODE, columnList);
+        isSameEntry &= isPropertyEqual(entry.getFinancialSubObjectCode(), KFSPropertyConstants.FINANCIAL_SUB_OBJECT_CODE, columnList);
+        isSameEntry &= isPropertyEqual(entry.getFinancialBalanceTypeCode(), KFSPropertyConstants.FINANCIAL_BALANCE_TYPE_CODE, columnList);
+        isSameEntry &= isPropertyEqual(entry.getFinancialObjectTypeCode(), KFSPropertyConstants.FINANCIAL_OBJECT_TYPE_CODE, columnList);
+        isSameEntry &= isPropertyEqual(entry.getUniversityFiscalPeriodCode(), KFSPropertyConstants.UNIVERSITY_FISCAL_PERIOD_CODE, columnList);
+        isSameEntry &= isPropertyEqual(entry.getFinancialDocumentTypeCode(), KFSPropertyConstants.FINANCIAL_DOCUMENT_TYPE_CODE, columnList);
+        isSameEntry &= isPropertyEqual(entry.getFinancialSystemOriginationCode(), KFSPropertyConstants.FINANCIAL_SYSTEM_ORIGINATION_CODE, columnList);
+        isSameEntry &= isPropertyEqual(entry.getDocumentNumber(), KFSPropertyConstants.DOCUMENT_NUMBER, columnList);
+        isSameEntry &= isPropertyEqual(entry.getTransactionLedgerEntrySequenceNumber().toString(), KFSPropertyConstants.TRANSACTION_ENTRY_SEQUENCE_NUMBER, columnList);
+
+        return isSameEntry;
+    }
+
+    public static void setFieldValue(ResultRow row, String field, String value) {
+        int col = getColumnIndexByProperty(row.getColumns(), field);
+        if (col != 1) {
+            Column column = row.getColumns().get(col);
+            if (value != null) {
+                column.setPropertyValue(value);
+            }
+        }
+    }
+
+    private static void setFieldInquiryURL(ResultRow row, String field) {
+        List<Column> columnList = row.getColumns();
+        int col = getColumnIndexByProperty(columnList, field);
+        if (col != -1) {
+            Column column = columnList.get(col);
+            if (StringUtils.isNotBlank(column.getPropertyValue())) {
+                Map<String, String> keys = getPrimaryKeysForInquiryURL(row, field);
+                if (keys.size() != 0) {
+                    String title = generateURLTitle(row, column, keys);
+                    String inquiryURL = generateInquiryURL(row, field, keys);
+
+                    HtmlData columnAnchor = new HtmlData.AnchorHtmlData(inquiryURL, title);
+                    column.setColumnAnchor(columnAnchor);
+                }
+            }
+        }
+    }
+
+    @SuppressWarnings("rawtypes")
+    private static Class getClassForField(String field) {
+        switch (field) {
+            case KFSPropertyConstants.GEC_DOCUMENT_NUMBER:
+            case KFSPropertyConstants.DOCUMENT_NUMBER:
+                return null;

```

Comment:
it's a trigger in another method, generateURLTitle(). Document numbers don't have titles, evidently.